static/page-templates/templates/vesta.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@iamtakashi  this template has links to `https://iamtakashinuxtest20200316221813.wordpress.com/wp-admin/post.php?post=33&amp;action=edit#` — is that expected?

Images are also referenced from `iamtakashinuxtest20200316221813.files.wordpress.com` — is that expected as well?

Reply:
I believe the content including images still need to be polished and optimised, and I don't think we've decided where to save the markup and the images. cc @ianstewart